ui/build.gradle

```diff
@@ -0,0 +1,100 @@
+apply from: '../versions.gradle'
+apply plugin: 'com.android.library'
+
+ext {
+    PUBLISH_GROUP_ID = 'com.xmartlabs'
+    PUBLISH_ARTIFACT_ID = 'ui'
+    PUBLISH_VERSION = '0.1'
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'com.jakewharton.butterknife'
+
+buildscript {
+    repositories {
+        jcenter()
+        maven { url 'https://maven.fabric.io/public' }
+    }
+    dependencies {
+        classpath "com.jakewharton:butterknife-gradle-plugin:8.2.1"
+        classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
```

Comment:
This is because this is in the buildscript, and when it gets executed, there's no "project". The 'buildscript' block gets evaluated before anything else from what i've researched.